src/frcnn_model.js

```diff
@@ -31,9 +31,19 @@ function CNTKFRCNNModel(opts) {
     this.cntkModelPath = opts.cntkModelPath;
     this.cntkPath = opts.cntkPath || DEFAULT_CNTK_INSTALL_PATH;
     this.cntkEnv = opts.cntkEnv;
+
+    var isWindows = process.platform == 'win32';
+    if (isWindows) {
+        this.anacondaPath = opts.anacondaPath || this.cntkPath;
```

Comment:
This is incorrect. You need to take the directory that CNTK resides in, in order to be consistent with the previous code.
E.g: this.anacondaPath = opts.anacondaPath || path.dirname(this.cntkPath);
(Make sure that 'path' is required in this file)